engine/glfw/lib/android/android_jni.h

```diff
@@ -0,0 +1,27 @@
+// Copyright 2020-2023 The Defold Foundation
+// Copyright 2014-2020 King
+// Copyright 2009-2014 Ragnar Svensson, Christian Murray
+// Licensed under the Defold License version 1.0 (the "License"); you may not use
+// this file except in compliance with the License.
+// 
+// You may obtain a copy of the License, together with FAQs at
+// https://www.defold.com/license
+// 
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+
+#ifndef _ANDROID_JNI_H_
+#define _ANDROID_JNI_H_
+
+#include "internal.h"
+#include <jni.h>
+
+extern struct android_app* g_AndroidApp;
```

Comment:
This belongs to a compile unit, e.g. android_jni.c

Reply:
The `g_AndroidApp` variable is used in several places,
here:
https://github.com/Fr1ar/defold/blob/6329dbb495efd1ed983de429df126f54bbd20a8c/engine/glfw/lib/android/android_joystick.c#L289

and here:
https://github.com/Fr1ar/defold/blob/6329dbb495efd1ed983de429df126f54bbd20a8c/engine/glfw/lib/android/android_util.c#L30

so it must be declared in the header file.

Of course, it could be moved to a cpp file and wrapped around with `android_app* getAndroidApp()` function, but I don't think that's necessary.
So what should I do, leave it as it is or wrap it around?